nuxeo-services/nuxeo-platform-filemanager-core/src/main/java/org/nuxeo/ecm/platform/filemanager/service/FileManagerService.java

```diff
@@ -269,6 +275,12 @@ public DocumentModel createDocumentFromBlob(CoreSession documentManager, Blob in
         return null;
     }
 
+    protected boolean isImporterAvailable(FileImporter importer, String normalizedMimeType, String mimeType,
+            boolean excludeOneToMany) {
+        return importer.isEnabled() && (!excludeOneToMany || !importer.isOneToMany())
```

Comment:
Personally I read this better with `&& !(importer.isOneToMany() && excludeOneToMany)` but up to you.